site2/docs/adaptors-kafka.md

```diff
@@ -129,7 +129,7 @@ Properties:
 | Config property                         | Supported | Notes                                                                         |
 |:----------------------------------------|:----------|:------------------------------------------------------------------------------|
 | `acks`                                  | Ignored   | Durability and quorum writes are configured at the namespace level            |
-| `auto.offset.reset`			  | Yes       | Will have a default value of 'latest' if user does not give specific setting. |
+| `auto.offset.reset`		 	          | Yes       | Will have a default value of `latest` if user does not give specific setting. |
```

Comment:
Added superfluous whitespace.

Reply:
I just fixed the `..` and aligned the columns in the source md text structure.

Reply:
they're not aligned though: https://raw.githubusercontent.com/apache/pulsar/09da58bcda81cba3d6b29331e7ce6884bd81f150/site2/docs/adaptors-kafka.md